content-src/experiments/activity-stream.yaml

```diff
@@ -116,7 +116,6 @@ contributors:
     display_name: 'Bryan Bell'
     title: 'Senior UX Designer'
     avatar: /static/images/experiments/activity-stream/avatars/bryan-bell.jpg
-installation_count: 46732
```

Comment:
:+1: These are an unused artifact since the days of exporting this data from the Django database